apps/hash/README.md

```diff
@@ -128,6 +128,16 @@ To run HASH locally, please follow these steps:
 
     See `package.json` → `scripts` for details and more options.
 
+1.  To develop using external integrations, you need a locally running Hashicorp Vault for secret storage
+
+    First, [install Vault](https://developer.hashicorp.com/vault/tutorials/getting-started/getting-started-install).
+
+    Then run a dev server:
+
+    ```sh
+    vault server -dev -dev-root-token-id=dev_root_token
+    ```
+
```

Comment:
Not sure we need this as it's part of external services

Reply:
Removed in https://github.com/hashintel/hash/pull/2796/commits/43e2b3ede58ec9f3fe936208a90bf2037c6d3c50